controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -87,7 +87,6 @@ void client_steering_task::work()
 
 void client_steering_task::steer_sta()
 {
-    Socket *sta_socket = database.get_node_socket(sta_mac);
```

Comment:
This one is different from the rest of the changes, so it should be mentioned explicitly in the commit message. Actually, I even think it could be a separate commit.

Reply:
It really not. This is used for backhaul roaming. Here we get the backhaul socket which is basically the same mac address of the parent bridge mac which we later get.
On line 144 I replace the validation of this socket by comparing the node type to `beerocks::TYPE_IRE_BACKHAUL`.
Since I remove all `get_node_socket` calls in this commit, this needs to be part of this commit/